frontend/src/service/StaticConfigDecision.service.ts

```diff
@@ -1,6 +1,9 @@
 import { IFeatureDecisionService } from './FeatureDecision.service';
 
 export class StaticConfigDecisionService implements IFeatureDecisionService {
+  includeSearchBar(): boolean {
+    return true;
```

Comment:
```suggestion
    return false;
```

This feature is not fully functioning yet. We want to hide it from users at the moment so that it's won't block other devs' deployment.